day7/will/index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Saturn</title>
+  <link rel="stylesheet" href="index.css" />
+  <link href="https://fonts.googleapis.com/css?family=Overpass+Mono:700&display=swap" rel="stylesheet" />
+  <script src="https://kit.fontawesome.com/a62b29f649.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <section class="mobileview-wrapper">
+    <img class="mobileview-img-phone" src="phone.png" alt="phonepic" />
+    <div class="mobileview-container">
+      <img src="logo.png" alt="logo" />
+      <p>Saturn is best experienced on our mobile app</p>
+    </div>
+    <button>Download Saturn</button>
+  </section>
+  <section class="tabletview-wrapper">
+    <div class="tabletview-title-container">
+      <img src="logo2.png" alt="logo" />
+      <div class="tabletview-title-block">
+        <div class="tabletview-title-circle">
+          <p>Period 2</p>
+          <h2>45 : 27</h2>
+          <h4>25% done</h4>
+        </div>
+        <div class="tabletview-title-text">
+          <div class="tabletview-title-classroom">
+            <h3>AP Computer Science</h3>
+            <p>Mr.Nessi • Room 25A • 9:30 - 10:43</p>
+          </div>
+          <div class="tabletview-title-progress">
+            <p><span>25%</span> of your day is completed!😎</p>
+            <div class="tabletview-progressbar">
+              <div></div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="tabletview-content-container">
+      <div class="tabletview-content-title">
+        <div class="tabletview-content-title-text">
+          <p>Belmont High School</p>
+          <h2>Monday, Feb. 24 - <span>A Day</span></h2>
+        </div>
+        <button><i class="fas fa-sign-out-alt"></i>Log Out</button>
+      </div>
+      <div class="tabletview-content-list-block">
+        <div class="tabletview-content-list">
```

Comment:
you are writing this twice for tablet and laptop, try to use only 1 set of html for those

Reply:
adjusted